README.md

```diff
@@ -16,20 +18,24 @@ Publish Laraberg files
 php artisan vendor:publish --provider="VanOns\Laraberg\LarabergServiceProvider"
 ```
 
-Laraberg provides a CSS file that should be present on the page you want to render content on:
+Laraberg provides a CSS file that should be present on the page you want to
+render content on:
 
 ```html
 <link rel="stylesheet" href="{{asset('vendor/laraberg/css/laraberg.css')}}">
 ```
-## Usage
+
+### Usage
 
 Simply register the field in your Resource
```

Comment:
Hier dus ook graag een `;` aan het eind omdat er een voorbeeld na komt (geldt ook voor de rest van de readme).

Reply:
Ik denk verder ook dat de teksten wat vloeiender kunnen. Zie als voorbeeld [hier](https://github.com/vanons/laravel-attachment-library/). Wat denk jij?

Reply:
Same as for Laraberg, I personally have no issues with the documentation.